### PR TITLE
updates to variables and subscription

### PR DESCRIPTION
**Pull request by cavonac**
previous notebook was not taking Subscription into account. Here, I'm setting to use the az context explicitly with the $Subscription variable

This PR fixes #

### CODE REVIEW

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb [hunk 1]
```diff
@@ -50,21 +51,22 @@
             "cell_type": "code",
             "source": [
                 "# Resource Group  \r\n",
-                "$ResourceGroupName  =   \"\"                             # Name of the resource group to create in the current subscription\r\n",
-                "$Location           =   \"\"                             # see Appendix for a list of location settings\r\n",
+                "$Subscription       = \"\" # Leave blank if \r\n",
```
Comment (Charles-Gagnon) [inline extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb:54]: Incomplete message here?

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb [hunk 2]
```diff
@@ -50,21 +51,22 @@
...
+                "$VMName             = \"\" # VM to create\r\n",
+                "$PublisherName      = \"\" # Name of Publisher, Default would be 'MicrosoftSQLServer'\r\n",
+                "$Version            = \"\" # Version of VM, Default would be 'latest'\r\n",
                 "\r\n",
                 "# Storage\r\n",
-                "$StorageAccountName =   $ResourceGroupName + \"_storage\"\r\n",
-                "$StorageSku         =   \"\"                             # Choose your storage sku (see appendix)\r\n",
-                "$StorageName        =   \"sqlstorage\" + (Get-Random -Minimum 1 -Maximum 100)\r\n",
+                "$StorageAccountName =   $ResourceGroup + \"_storage\"\r\n",
+                "$StorageSku         = \"\"  # Choose your storage sku (see appendix)\r\n",
```
Comment (Charles-Gagnon) [inline extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb:65]: SKU is usually capitalized

**Review by Charles-Gagnon — APPROVED** [review]

**Review by smartguest — APPROVED** [review]
Looks fine aside from the issues that Charles mentioned

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb (hunk 1 above)
Comment (smartguest) [inline extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb:54]: Same here.

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb (hunk 2 above)
Comment (smartguest) [inline extensions/azurehybridtoolkit/notebooks/hybridbook/content/provisioning/create-sqlvm.ipynb:65]: same here

**Review by smartguest — APPROVED** [review]

**Pull request merged**